packages/code-du-travail-frontend/src/modules/simulateur-embauche/index.tsx

```diff
@@ -0,0 +1,98 @@
+"use client";
+import { Highlight } from "@codegouvfr/react-dsfr/Highlight";
+import { fr } from "@codegouvfr/react-dsfr";
+import { useEffect, createRef, useState } from "react";
+import { ContainerMiddle } from "../layout/ContainerMiddle";
+import { RelatedItem } from "../documents";
+import { Feedback } from "../layout/feedback";
+
+type Props = {
+  relatedItems: {
+    items: RelatedItem[];
+    title: string;
+  }[];
+};
+
+export const HiringSimulator = ({ relatedItems }: Props) => {
+  const simRef = createRef<HTMLDivElement>();
+  const [state, setState] = useState({
+    error: "",
+    simulator: "loading",
+  });
+  const onError = (error) => {
+    setState({ error, simulator: "error" });
+  };
+
+  const onLoad = () => {
+    setState({ simulator: "success", error: "" });
+    if (
+      !simRef.current ||
+      !simRef.current.querySelector("#simulateurEmbauche")
+    ) {
+      setState({ error: "empty child", simulator: "error" });
+    }
+  };
+  const useScript = () => {
+    useEffect(() => {
+      const script = document.createElement("script");
+
+      script.src =
+        "https://mon-entreprise.urssaf.fr/simulateur-iframe-integration.js";
+      script.async = true;
+      script.dataset.couleur = "#2975D1";
```

Comment:
peut être qu'on met le bleu mariane ici ?
```suggestion
      script.dataset.couleur = "#000091";
```